### PR TITLE
Remove collapsibility and change names implying collapsibility

### PR DESCRIPTION
**Pull request by Erin007**
Initially, the Teacher Homepage was designed with collapsible sections; however, we've decided the sections no longer need to be collapsible. There was a temporary change to have CollapsibleSection components toggle between being collapsible or not.  Now, given that we don't have any intention in the foreseeable future to collapse sections, this PR replaces CollapsibleSection with ContentBox and renames other components that used CollapsibleSections.

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @tanyaparker

File: apps/src/templates/studioHomepages/ContentBox.jsx [hunk 1]
```diff
@@ -48,7 +49,7 @@ const styles = {
   }
 };
 
-const CollapsibleSection = React.createClass({
+const ContentBox= React.createClass({
```
Comment (Bjvanminnen) [inline apps/src/templates/studioHomepages/ContentBox.jsx]: This name is now fairly generic. It's likely worth having a comment describing what makes something a ContentBox (i.e. what this component is and when it should/should not be used).

File: apps/src/templates/studioHomepages/ContentBox.jsx [hunk 2]
```diff
@@ -104,11 +76,18 @@ const CollapsibleSection = React.createClass({
...
         </div>
-        {this.renderContent()}
+        {React.Children.map(this.props.children, (child, index) => {
+          return (
+            <div key={index}>
+              {child}
+              {(index % 2 === 0) && <div style={styles.spacer}>.</div>}
+            </div>
+          );
+        })}
```
Comment (Bjvanminnen) [inline apps/src/templates/studioHomepages/ContentBox.jsx]: note: you could simplify this to
```jsx
React.Children.map(this.props.children, (child, index) => (
  <div>stuff inside your div</div>
));
```

File: apps/src/templates/studioHomepages/ContentBox.story.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+const exampleCard = {
+  name: "CSP Unit 2 - Digital Information",
+  description: "Explore how more complex digital information is represented and manipulated through computation and visualization",
+  image: "this is where there will be the source for the photo",
+  link: "link to the course",
+  assignedSections: []
+};
+
+// ContentBox is a generic component that will render whatever child components are passed to it.  CourseCards are used here as an example because it was first built to render them for the Teacher Homepage.
```
Comment (Bjvanminnen) [inline apps/src/templates/studioHomepages/ContentBox.story.jsx]: nit: lines should wrap at 80 chars

**Comment by Bjvanminnen** [comment]
👍 Big fan of renaming things as their purpose becomes more clear :)

File: apps/src/templates/studioHomepages/ContentBox.jsx (hunk 1 above)
Comment (breville) [reply apps/src/templates/studioHomepages/ContentBox.jsx]: Box implies a visual box.  I'd recommend a name ending in Container to better express what this does (unless it really draws a box?)

**Pull request merged**